chain/chain/src/chain.rs

```diff
@@ -508,6 +508,14 @@ impl Chain {
         });
     }
 
+    fn save_block_height_processed(&mut self, block_height: BlockHeight) -> Result<(), Error> {
```

Comment:
It doesn't really need to be in storage. Adding it to storage adds all sorts of possible config issues with GCing.
We only add to the collection if the height is in one of the known epochs, so can just use an in-memory unbounded hashtable.

Reply:
Discussed offline. Looks like storage is fine.